app/io/pathfinder/websockets/WebSocketActor.scala

```diff
@@ -1,9 +1,15 @@
 package io.pathfinder.websockets
 
 import akka.actor.{Props, Actor, ActorRef}
-import io.pathfinder.websockets.ModelTypes._
+import io.pathfinder.models.{Vehicle, Commodity}
+import io.pathfinder.routing.Router
+import io.pathfinder.websockets.ModelTypes.ModelType
+import io.pathfinder.websockets.controllers.VehicleSocketController._
+import play.Logger
 import play.api.libs.concurrent.Execution.Implicits.defaultContext
 import io.pathfinder.websockets.controllers.{CommoditySocketController, ClusterSocketController, WebSocketController, VehicleSocketController}
+import scala.concurrent.ExecutionContext.Implicits.global
+import play.api.libs.iteratee.{Iteratee, Concurrent}
 
 object WebSocketActor {
```

Comment:
Why is this an object and not a class? If all of our web socket requests are routed through a singleton, we are limiting our scaling potential